main.cpp

```diff
@@ -0,0 +1,28 @@
+#include <iostream>
+#include <iomanip>
+
+int main() {
+
+    const int SHARES_BOUGHT = 1000;
+    const double PURCHASE_PRICE = 45.50;
+    const double COMMISSION_RATE = .02;
+    const int SHARES_SOLD = 1000;
+    const double SALE_PRICE = 56.90;
+    double amount_paid = SHARES_BOUGHT * PURCHASE_PRICE;
+    double first_commission = amount_paid * COMMISSION_RATE;
+    double amount_sold = SHARES_SOLD * SALE_PRICE;
+    double second_commission = amount_sold * COMMISSION_RATE;
```

Comment:
You should consider a more descriptive name for these commission numbers as opposed to `first_` and `second_`
possibly `purchase_` and `sale_` `commission`. Unfortunately first and second are ambiguous in this case. When variable names are ambiguous they also suffer from a lack of meaning.